src/isar_turtlebot/inspection_pose.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
```

Comment:
This file should probably be put into a utility folder and not be on the same level as robot_interface